.github/workflows/main.yml

```
@@ -1,19 +1,19 @@
-    branches: [master, testnet]
-
-jobs:
-  Jest:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - name: Run Tests
-        run: |
-          npm install --legacy-peer-deps
-          npm test
+name: Unit Tests
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)